public/pages/DetectorsList/utils/tableUtils.tsx

```diff
@@ -28,26 +28,98 @@ const renderTime = (time: number) => {
   return DEFAULT_EMPTY_DATA;
 };
 
+const renderLastUpdateTime = (lastUpdateTime: number, detector: Detector) => {
+  const momentTime = moment(detector.lastUpdateTime);
+  if (detector.lastUpdateTime && momentTime.isValid()) return momentTime.format('MM/DD/YY h:mm A');
+  return DEFAULT_EMPTY_DATA;
+};
+
+const renderIndices = (indices: string[], detector: Detector) => {
+  if (detector == null || detector.indices == null || detector.indices.length == 0) return DEFAULT_EMPTY_DATA;
```

Comment:
use get from `lodash`. You can find example here.

https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/cf231c238ec505223fe06d66ec02787a3d8cec59/public/pages/PreviewDetector/containers/utils/featuresToFormik.ts#L27-L31

Reply:
After revisiting this function I realized the only thing being passed to render indices was a string[]. I removed the unnecessary Detector param entirely